docs/faq.md

```diff
@@ -0,0 +1,204 @@
+`DbProviderFactories`, `DbConnection`, `IDbConnection` etc) as well as the the
+general ADO.NET facilities (`DataSet`, `DataTable` etc) APIs.
+
+We don't plan on adding any specific providers to .NET Standard as their
+applicability varies (e.g. it's a highly unlikely scenario to use the SQL Server
+client from an iOS device, but it would make sense to use a provider that can
+store data on the device, such as SQLite). The expectation is that those sit on
+top of .NET Standard or remain platform-specific.
+
+## Why is JSON.NET not part of .NET Standard?
+
+Today, the best library for dealing with JSON is JSON.NET. But by adding it to
+the .NET Standard we'd do the community a disservice. What matters is that the
+JSON support is widely available. And James, the author of JSON.NET, does a
+great job making sure that JSON.NET is available everywhere. His ability to do
+this successfully is a function of how hard it is for him to make changes. The
+best way to do this is by creating a library that targets .NET Standard because
+it can be updated independently from the standard itself and everyone
+immediately benefits.
+
+Of course, this doesn't mean we can't or shouldn't provide some built-in JSON
+support. We've talked with James about this in the past and I believe there is a
+lot of opportunity for us to collaborate with him on an even more performant way
+to provide JSON support in .NET. However, we're very interested in doing this
+with him rather than just building "another" JSON.NET. We want a strong
+ecosystem for .NET, but this can only happen if we embrace libraries based on
+merit, rather than by who wrote it. That's what open source is all about.
+
+## Why do you include APIs that don't work everywhere?
+
+We generally don't include APIs in .NET Standard that don't work everywhere, and
+instead provide them as libraries that sit above .NET Standard.
+
+But if you think about it: we can't make *type members* (e.g. methods,
+properties, etc) additive. The only thing you can make additive are *types*, as
+two different types can live in separate assemblies but we don't have a
+mechanism to split a single type across two different assemblies. In those
+cases, we leave the members on the type and let platforms that cannot
+meaningfully implement them throw `PlatformNotSupportedException`.
+
+Moving forward, we try to avoid creating types where only parts of it work
+everywhere. But as always, there will be cases where we couldn't predict the
+future and are forced to throw.
+
+## Will there be tooling to highlight APIs that don't work everywhere?
+
+Our current focus is on providing APIs either as part of .NET Standard or as
+independent packages that sit on top of .NET Standard. In some cases, certain
+APIs will not be supported everywhere and throw `PlatformNotSupportedException`.
+While that isn't ideal, it's much simpler than the alternatives, which are:
+
+* Using `#if`, i.e. cross-compile
```

Comment:
i.e. : Don't use. Use that is instead. (MSFT style guide) although I'm not completely sure this would be the right meaning here